pymeasure/display/browser.py

```diff
@@ -34,7 +34,20 @@
 log.addHandler(logging.NullHandler())
 
 
-class BrowserItem(QtWidgets.QTreeWidgetItem):
+class BaseBrowserItem(object):
+    status_label = {
```

Comment:
I think some docstring is needed here.

Reply:
I've added a docstring placeholder at the moment, but I think it requires more explanation. 
The reason for this base class is that the `Experiment` class in `pymeasure/display/manager.py ` has an attribute [`browser_item`](https://github.com/mcdo0486/pymeasure/blob/bafb2b3ae37b2feeccbf560acc468c05bbfb8d6d/pymeasure/display/manager.py#L56) which is used to display the [GUI progress bar](https://github.com/mcdo0486/pymeasure/blob/bafb2b3ae37b2feeccbf560acc468c05bbfb8d6d/pymeasure/display/browser.py#L50) in the experiment queue table.

To keep the amount of code changes to a minimum, I created a base class that shares a dictionary of procedure statuses and the two names of the two qt methods, hence the camelcase, and kept the attribute name.

`Experiment.browser_item`, now that it is used to display progress in both GUI mode and console mode (which doesn't have a browser), should probably be renamed to `Experiment.progress` or something. And `setProgress` and `setStatus` don't need to be in camelcase, but that is the norm for the gui qt classes.

Reply:
Ok, thanks. I have incorporated your changes in the my fork's branch.
I did not merge this pull request, since I used mainly changes from `browser.py`, `manager.py`, but I did some other changes in `example/Basic/console.py` and `pymeasure/display/console.py`.
Would you mind having a look at my latest changes and see if the key point of this PR has been incorporated ?

Reply:
Yup, the key point has been incorporated, thanks for considering the changes